src/app/legacy/components/RadioSchedule/ScheduleItemHeader/index.jsx

```diff
@@ -85,40 +73,35 @@ const ScheduleItemHeader = ({
     ['media', 'listenNext'],
     translations,
   );
-
   const listenLabelTranslations = {
     live: listenLive,
     next: listenNext,
     onDemand: listen,
   };
-
   const formattedStartTime = formatUnixTimestamp({
     timestamp: startTime,
     format: 'HH:mm',
     timezone,
     locale,
     isRelative: false,
   });
-
   const formattedDuration = detokenise(
     durationLabel,
     durationDictionary({ duration, locale }),
   );
-
   const episodeTitle = formatUnixTimestamp({
     timestamp: startTime,
     format: 'LL',
     timezone,
     locale,
     isRelative: false,
   });
-
   const content = (
     // This is a temporary fix for the a11y nested span's bug experienced in TalkBack, refer to the following issue: https://github.com/bbc/simorgh/issues/9652
     // eslint-disable-next-line jsx-a11y/aria-role
     <span role="text" id={`scheduleItem-${id}`}>
       <VisuallyHiddenText>{`${listenLabelTranslations[state]}, `}</VisuallyHiddenText>
-      {isLive && <LiveLabel liveText={liveLabel} ariaHidden />}
+      {isLive && <LiveLabel id={`radio-schedule-${id}`} />}
```

Comment:
I've spotted a potential issue on Storybook with the radio schedule & how it reads out "Listen Live" - https://wsteama-831-refactor-livelabel--5d28eb3fe163f6002046d6fa.chromatic.com/?path=/story/components-radio-schedule--radio-schedule&knob-Select%20a%20service=afrique - let's check how this works on latest storybook... 🤔

Reply:
Yes, as suspected, there's duplicate "Listen Live" and "Live" screenreader text... I think we need to pass in "Listen Live" as the offscreen text for the component, and do not add the VisuallyHiddenText if it's live... let's have a look at the best way to fix this

Reply:
cc @greenc05 we spotted the issue with the radio schedule and will aim to fix.

Reply:
cc @LilyL0u this seems to work https://github.com/bbc/simorgh/commit/755436fa9659f1a90d738787bebc5faf93662e17 - you would need to cherrypick this commit onto your branch

```
git cherry-pick 755436fa9659f1a90d738787bebc5faf93662e17
```

Reply:
More work was done after this to make it work